minio_storage/management/commands/minio.py

```diff
@@ -122,10 +122,11 @@ def handle(self, *args, **options):
         raise CommandError("command name required")
 
     def storage(self, options):
-        class_name = {
-            "media": "minio_storage.storage.MinioMediaStorage",
-            "static": "minio_storage.storage.MinioStaticStorage",
-        }.get(options["class"], options["class"])
+        class_name: str = options["class"]
+        if class_name == "media":
+            class_name = "minio_storage.storage.MinioMediaStorage"
+        elif class_name == "static":
+            class_name = "minio_storage.storage.MinioStaticStorage"
```

Comment:
It's probably possible to get the correct type annotation with the old implementation, but this seems a lot easier to understand at first glance.